examples/with_service_broker_rhel.yaml

```diff
@@ -0,0 +1,23 @@
+apiVersion: "app.redislabs.com/v1alpha1"
+kind: "RedisEnterpriseCluster"
+metadata:
+  name: "redis-enterprise-cluster"
+spec:
+  nodes: 3
+  persistentSpec:
+    enabled: true
+    storageClassName: "gp2"
+  serviceBrokerSpec:
+    enabled: true
+    persistentSpec:
+    storageClassName: "gp2"
+    imageSpec:
+     imagePullPolicy:  Always
+     repository:       redislabs/service-broker-internal
+     versionTag:       39_5bfe271.rhel7
+    redisEnterpriseImageSpec:
```

Comment:
1. wrong indentation, it needs to be at the same level as `serviceBrokerSpec`.
2. missing:
```
  redisEnterpriseControllerImageSpec:
    imagePullPolicy:  IfNotPresent
    Repository:       redislabs/k8s-controller
    versionTag:       65_7e696a1.rhel7
```